test/e2e/sample_e2e_test.go

```diff
@@ -114,12 +114,11 @@ var _ = Describe("Sample", func() {
 
 			// TODO: do we need to create catalog images are each scenario ? can we move building images to once per test suite (group of tests that uses the catalog)
 			// don't want to rebuild the image if it already exists
-			operatorImage := "quay.io/cdjohnson/ibm-sample-panamax-operator@sha256:47c9fcef261f3f26dbb6c05f904d11563aae43f61d72e0e4556399f3003c97d0"
```

Comment:
I am not able use panamax operator, because when panamax operator starts up, its expecting panamax CRD which we are not creating the tests

it fails with 
```
{"level":"error","ts":1615913888.660425,"logger":"controller-runtime.source","msg":"if kind is a CRD, it should be installed before calling Start","kind":"Panamax.sample.ibm.com","error":"no matches for kind \"Panamax\" in version \"sample.ibm.com/v1beta1\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/root/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start\n\t/root/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.0/pkg/source/source.go:105\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\t/root/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.0/pkg/internal/controller/controller.go:165\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start\n\t/root/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.0/pkg/internal/controller/controller.go:198\nsigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startLeaderElectionRunnables.func1\n\t/root/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.6.0/pkg/manager/internal.go:514"}
{"level":"error","ts":1615913888.6608453,"logger":"cmd","msg":"Manager exited non-zero","error":"no matches for kind \"Panamax\" in version \"sample.ibm.com/v1beta1\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\t/root/go/pkg/mod/github.com/go-logr/zapr@v0.1.1/zapr.go:128\nmain.main\n\tibm-sample-panamax-operator/cmd/manager/main.go:153\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203"}
```

Reply:
I changed this